Furball.Game/Screens/BasicTestScreen.cs

```diff
@@ -28,6 +31,37 @@ public class BasicTestScreen : Screen {
 
             this.Manager.Add(whiteTexture);
 
+            UiButtonDrawable testButton = new("test button", ContentReader.LoadRawAsset("default-font.ttf"), 50, Color.White, Color.Black) {
+                Position      = new Vector2(200, 200),
+                ColorOverride = Color.Blue
+            };
+            testButton.OnHover += delegate {
+                testButton.Tweens.Add(
+                new VectorTween(
+                TweenType.Movement,
+                testButton.Position,
+                new(testButton.Position.X + FurballGame.Random.NextSingle(-50, 50), testButton.Position.Y + FurballGame.Random.NextSingle(-50, 50)),
+                FurballGame.Time,
+                FurballGame.Time + 1000
+                )
+                );
+            };
+            testButton.OnUnHover += delegate {
```

Comment:
Lets maybe call this OnHoverLost

Reply:
not related to this PR in general so ill do it outside of the PR after this is merged to prevent conflicts